eng/emitter-package.json

```diff
@@ -2,16 +2,16 @@
   "name": "typescript-emitter-package",
   "main": "dist/src/index.js",
   "dependencies": {
-    "@azure-tools/typespec-ts": "0.27.0",
+    "@azure-tools/typespec-ts": "0.28.0",
     "@azure-tools/typespec-azure-core": "0.41.0",
-    "@azure-tools/typespec-autorest": "0.41.0",
-    "@azure-tools/typespec-client-generator-core": "0.42.0-dev.6",
+    "@azure-tools/typespec-autorest": "0.41.1",
+    "@azure-tools/typespec-client-generator-core": "0.41.7",
```

Comment:
should be `0.41.5`, https://github.com/Azure/autorest.typescript/blob/main/packages/typespec-ts/package.json#L91